source/server-discovery-and-monitoring/server-discovery-and-monitoring.md

```diff
@@ -226,7 +226,9 @@ Fields:
     field in the server's hello or legacy hello response, in the case that the server reports an address different from
     the address the client uses.
 
-- (=) `error`: information about the last error related to this server. Default null.
+- (=) `error`: information about the last error related to this server. Default null. MUST contain or be able to produce
+    a string describing the error. The name of the field containing the string describing the error SHOULD be what is
+    most idiomatic for each driver.
```

Comment:
What was the motivation for adding "The name of the field containing the string describing the error SHOULD be what is most idiomatic for each driver."?

Reply:
I was thinking that the `error` field would be an object representing an error and since different languages have different ways of doing that, I think it's better to defer to driver engineers on what is the best way to communicate that to users. In Node we'd likely do that with a new `MongoError` subclass (which itself subclasses the native Node `Error` type) that uses a `message` field to hold human readable information about the error. We currently have it typed as a generic `MongoError`

Reply:
I see. I think we can remove that sentence since it generally always true of any spec prescribed api.